pkgs/by-name/hy/hydra/package.nix

```diff
@@ -119,30 +119,30 @@ let
         UUID4Tiny
         XMLSimple
         YAML
-        nix.perl-bindings
+        nix.libs.nix-perl-bindings
         git
       ];
   };
 
-  nix-eval-jobs' = nix-eval-jobs.overrideAttrs (_: {
-    version = "2.25.0-unstable-2025-02-13";
+  nix-eval-jobs' = (nix-eval-jobs.override { inherit nix; }).overrideAttrs (_: {
```

Comment:
Why do the Nix versions need to be consistent now? To my understanding, since #364142, `nix-eval-jobs` has been on 2.25 and Hydra has been on 2.24. Does this mean there are issues in `nixos-unstable`?

Reply:
It also looks like `nix-eval-jobs`'s Nix version should not be changed if possible: https://github.com/NixOS/nixpkgs/blob/d74a2335ac9c133d6bbec9fc98d91a77f1604c1f/pkgs/tools/package-management/nix-eval-jobs/default.nix#L37-L40